NixSupport/nixosModules/services/worker.nix

```diff
@@ -0,0 +1,29 @@
+{ config, pkgs, ihpApp, lib, ... }:
+let
+    cfg = config.services.ihp;
+in
+{
+    systemd.services.worker = {
+        enable = true;
+        after = [ "network.target" ];
+        wantedBy = [ "multi-user.target" ];
+        serviceConfig = {
+            Type = "simple";
+            Restart = "always";
+            WorkingDirectory = "${ihpApp}/lib";
+            ExecStart = "${ihpApp}/bin/RunJobs";
+        };
+        environment =
+            let
+                defaultEnv = {
+                    PORT = "${toString cfg.appPort}";
+                    IHP_ENV = cfg.ihpEnv;
+                    IHP_BASEURL = cfg.baseUrl;
+                    IHP_REQUEST_LOGGER_IP_ADDR_SOURCE = cfg.requestLoggerIPAddrSource;
+                    DATABASE_RUL = cfg.databaseUrl;
+                    IHP_SESSION_SECRET = cfg.sessionSecret;
+                };
+            in
+                defaultEnv // cfg.additionalEnvVars;
```

Comment:
Could the systemd services also support environment files so secrets are not stored in version control or in the nix store?

Reply:
👍